src/vs/workbench/contrib/extensions/browser/extensionsViews.ts

```diff
@@ -423,6 +423,12 @@ export class ExtensionsListView extends ViewletPanel {
 			options.sortBy = SortBy.InstallCount;
 		}
 
+		// {{SQL CARBON EDIT}}
+		if (/@visualizerExtensions/i.test(query.value)) {
+			return this.getVisualizerExtensions(token);
+		}
+		// End of {{SQL CARBON EDIT}}
```

Comment:
please reformat to `// {{SQL CARBON EDIT}} - ...` the beginning of the tag should be consistent to help with searching the source